### DISCUSSION TITLE
Kratos Mesh

### DISCUSSION THREAD

**Discussion by maxnezdyur** [KratosMultiphysics/Kratos#10736] [General]
I saw some other discussions about this but still wasn't sure what is the best option. For creating a complex mesh, what do most people use that allows users to put BCs in a .mdpa format?

**Comment by loumalouomega** [comment]
Usually people use @KratosMultiphysics/gidproblemtype

**Comment by rubenzorrilla** [comment]
There's also a Salome GUI (I point to @philbucher for further details about this) as well as a basic implementation in [meshio](https://github.com/nschloe/meshio).

**Reply by philbucher** [reply]
You can check our salome plugin: https://github.com/KratosMultiphysics/KratosSalomePlugin

**Reply by maxnezdyur** [reply]
This is perfect. I don't know how I missed it. 
Thank you!

**Comment by escolano** [comment]
About using Kratos in GiD, it is only necessary to install GiD [https://www.gidsimulation.com/gid-for-science/downloads]
and go to menu Data->Problemtypes->Internet retrieve... and select Kratos.
The Kratos problemtype and Kratos already compiled for this platform (Linux, Windows, or some old macOs version) will be get, and appear to be load in the mensu Data->Problemtypes->Kratos
Then can assign create or import the geometry, assign conditions, and directly calculate with Kratos, or only export its input files to be calculated from a console
